VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -44,6 +50,21 @@ public void setAppNameContext() {
             this.properties.put(Keys.CONTEXT, getAppNameContext());
         }
 
+        public void setCustomDimensions() {
```

Comment:
Actually, since this method seems to only be called from the constructor, we might as well port the logic there directly I guess.